.github/workflows/coverage-tests.yml

```diff
@@ -24,19 +28,19 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: 'true'
-    - uses: mamba-org/provision-with-micromamba@v14
+    - uses: mamba-org/setup-micromamba@v1.8.1
```

Comment:
Would recommend `conda-incubator/setup-miniconda@v3`

This is used successfully in a bunch of places at this point

Ref: https://github.com/conda-incubator/setup-miniconda